research/picai/nnunet_scripts/predict.py

```diff
@@ -0,0 +1,58 @@
+import argparse
+
+from nnunetv2.inference.predict_from_raw_data import nnUNetPredictor
+
+
+def main() -> None:
```

Comment:
Similar to the last PR, can you could add a section to the README with a description of the purpose of the script, what it does and an example invocation

Reply:
for the predict and eval script